creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.ts

```diff
@@ -623,7 +623,7 @@ const _additionalSyncIsRequired = async (
     }
   }
 
-  if (syncStatus.startsWith('success')) {
+  if (syncStatus?.startsWith('success')) {
```

Comment:
if syncStatus is not defined then we enter the else case which immediately records a failure - is that correct? or is there a valid third state where sync status is undefined

Reply:
that's correct. if it's undefined then it was a failure because the secondary didn't return any completion status so that means it probably didn't process it or ran into an error before saving what the result was in redis

Reply:
kk - we're rethinking this anyway so all good